### ISSUE TITLE
Core dump during training with contrastive augmentation

### ISSUE THREAD

**Issue by CdAB63** [AlexeyAB/darknet#7128]
Wich version? Git pull on 2020-12-14
OS System: Ubuntu 20.04.1 - Intel computer (I7 4C8T), 32GBytes DDR4 RAM, NVIDIA 1060 (6GBytes VRAM), 3TB storage
Cuda: 10.2 + cudnn 7.6 (old but I have to meet ends with tensorflow-gpu)
Built with: ./build.sh
In the configuration file the contrastive training disabled (commented out) and the contrastive on
Dataset size: 48000 images, test set 588 images
Training procedure:
1st 60000 iterations: disable both the contrastive and adversarial augmentaitions in cfg file and train with:
./darknet detector train cfg/weapon.data cfg/weapon-y4tiny-contrastive.cfg -map
After 60000 iterations: enable contrastive augmentation in .cfg file and train with:
./darknet detector train cfg/weapon.data cfg/weapon-y4tiny-contrastive.cfg backup/weapon-y4tiny-contrastive_60000.weights -map

After a while (some minutes) there's a core dump.

 (next mAP calculation at 66260 iterations) 
 60187: 0.359779, 0.527124 avg loss, 0.002610 rate, 1.736301 seconds, 1925984 images, 278.178517 hours left
  avg_contrastive_acc = 40.932590 
Loaded: 0.000063 seconds
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 52 Avg (IOU: 0.755208), count: 14, class_loss = 1.748249, iou_loss = 2.002278, total_loss = 3.750527 
 Contrast accuracy = 42.000000 %, all = 14, good = 6, same = 14, diff = 14 
 contrastive loss = 1.432542 

 total_bbox = 3830, rewritten_bbox = 0.000000 % 
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 52 Avg (IOU: 0.752656), count: 6, class_loss = 0.770700, iou_loss = 0.076087, total_loss = 0.846787 
 Contrast accuracy = 33.000000 %, all = 6, good = 2, same = 6, diff = 6 
Falha de segmentação (imagem do núcleo gravada)
cdab63@Predator:AlexeyAB$ 

[weapon-y4tiny-contrastive.txt](https://github.com/AlexeyAB/darknet/files/5691635/weapon-y4tiny-contrastive.txt)

```
